.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    name: PHP v${{ matrix.php }} with phpstan ${{ matrix.phpstan }} and code_standart ${{ matrix.code_standart}}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        include:
+        - { os: ubuntu-latest, php: 7.1, experimental: false}
+        - { os: ubuntu-latest, php: 7.1, phpstan: true, experimental: false}
+        - { os: ubuntu-latest, php: 7.2, experimental: false}
+        - { os: ubuntu-latest, php: 7.2, code_standart: true, experimental: false}
+        - { os: ubuntu-latest, php: 7.3, experimental: false}
+        - { os: ubuntu-latest, php: 7.4, experimental: false}
+        - { os: ubuntu-latest, php: 8.0, experimental: true}
```

Comment:
```suggestion
        - { os: ubuntu-latest, php: 8.0, experimental: false}
```
Or maybe just drop the experimental flag entirely.

Reply:
https://github.com/Smolevich/FastRoute/runs/1585187769?check_suite_focus=true